build.gradle

```diff
@@ -11,6 +11,11 @@ ext {
     apacheCommonsCliVersion = '1.4'
     jsonVersion = '20180130'
     slf4jVersion = '1.7.25'
+    restAssuredVersion = '4.1.1'
+    restAssuredKotlinExtVersion = '4.1.2'
+    junitJupiterVersion = '5.5.2'
+    mockkVersion = '1.9.3'
+    jacksonModuleKotlinVersion = '2.10.0.pr3'
```

Comment:
The latest version seems to be `2.10.0` (without the `pr` suffix):
https://mvnrepository.com/artifact/com.fasterxml.jackson.module/jackson-module-kotlin/2.10.0

Reply:
Changed to 2.10.0